common/common.go

```diff
@@ -329,6 +347,17 @@ func WithNetLogstore(lt LogstoreType) NetOption {
 	}
 }
 
+func WithNetPulling(disabled bool, threadLimit int, startAfter, initialInterval, interval time.Duration) NetOption {
```

Comment:
It's a little surprising that `WithNetPulling(true, ...)` actually disables net pulling. I'd even vote for removing the `disabled` parameter entirely, so calling this simply enables pulling with the params.

Reply:
If pulling should be on by default, we could provide another utility called `NoNetPulling()` to disable it.

Reply:
Yeah, that is pretty weird now that you mention it :) I'll go with the `NoNetPulling` option.